week-2/G-conditionals-2/exercise-2.js

```diff
@@ -8,7 +8,11 @@
 */
 
 function studentPassed(grade) {
-
+  if (grade >= 50) {
+    return (grade = "passed");
```

Comment:
Do not mutate (re-assign) the parameter passed to the function, as this will change it outside the function too.

`return "passed"` is sufficient here.